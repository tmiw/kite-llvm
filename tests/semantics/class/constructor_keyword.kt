class X [
    construct() [ this.elite = 1337|obj; 1; ];
    construct(val) [ this.elite = val|obj; 1; ];
];

x = make X();
x.elite|print;
x = make X(1234);
x.elite|print;
