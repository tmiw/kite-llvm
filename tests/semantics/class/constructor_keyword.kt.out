1337
1234
